src/core/js/views/drawerView.js

```diff
@@ -86,7 +86,14 @@ define([
             this._isCustomViewVisible = true;
             Adapt.trigger('drawer:empty');
             this.showDrawer();
+
             this.$('.drawer-holder').html(view);
+
+            var model = this.collection.findWhere({
```

Comment:
Sorry, I'm not fully understanding your issue; would you mind elaborating a bit more?

I tested this PR with both [Resources](https://github.com/adaptlearning/adapt-contrib-resources) and [Glossary](https://github.com/adaptlearning/adapt-contrib-glossary) to check that the title switches out correctly.